src/app/shared/recognize-video-type-id.ts

```diff
@@ -55,39 +44,35 @@ export function getVideoId(str: any) {
   }
   return metadata;
 }
-
 /**
  * Get the vimeo id.
- * @param {string} str - the url from which you want to extract the id
- * @returns {string|undefined}
+ * @param; {string} str - the url from which you want to extract the id
+ * @returns; {string|undefined}
  */
```

Comment:
@param i @returns często się pojawiają a nie mogłem nigdzie znaleźć o tym informacji. Raczej usunąć bo są wplecione w komentarz i sensu przynajmniej ja dla nich nie widzę.

Reply:
to są jsdocsy, sposób dokumentowania kodu - dzięki temu opisowi w wielu edytorach, np w vsc, po najechaniu na funkcję wyświetli Ci się opis co dany parameter oznacza. Nie musisz ich robić ale jak sa to nie warto usuwać. Tutaj możesz poczytać https://jsdoc.app/tags-param.html